### ISSUE TITLE
Clarifying update procedure

### ISSUE THREAD

**Issue by almostEric** [VCVRack/community#425]
My 0.6.1 never got built, and now I have a 0.6.2 release. I understand that each developer gets one "issue" to notify library team. My question is do I just update my original post, or do I tack it on the end?

**Comment by phdsg** [comment]
i think i get notifications either way.
maybe looks cleaner if the up-to-date info is always on top.

**Comment by AndrewBelt** [comment]
I'd say post a new comment in your plugin thread with the build information just to keep a history.

**Comment by AndrewBelt** [comment]
It's fine until it actually starts slowing your browser down. Since we won't be posting images, that would happen aroud 300-500 posts, in which case you could simply close the issue and create a new one with the same title.

**Comment by phdsg** [comment]
also i believe that older content gets folded away in longer threads...

**Comment by AndrewBelt** [comment]
Yeah, I think that's a brand new GitHub feature.

**Comment by almostEric** [comment]
ok, well for 0.6.2 I just updated my original post, since it hasn't been release yet anyway. I'll tack on at end going forward. Can the update plugin instructions be, um, updated? I think making everything as clear as possible for developers goes a long way

**Comment by AndrewBelt** [comment]
Edited in c11f14d. Let me know if this is not what you meant by "update plugin instructions".

**Comment by almostEric** [comment]
Much better

**Comment by phdsg** [comment]
@AndrewBelt the website needs an edit too (https://vcvrack.com/plugins.html). at the bottom it says:
> Developers: Submit new plugins and updates to https://github.com/VCVRack/community by sending a pull request.

**Comment by almostEric** [comment]
So, are the builds automated now, or is it still initiated manually?

**Comment by AndrewBelt** [comment]
They'll always be kicked off manually, but everything's automated after that.

**Comment by almostEric** [comment]
Are these builds going to be done on regular basis? My new version has been sitting unreleased for 4 days now, I see some other folks that have been waiting longer. If it is weekly, bi-weekly, whatever, just want to know what to expect.

**Comment by AndrewBelt** [comment]
Yes, eventually. I'm trying to ramp up to once a day, but life responsibilities are getting in the way.
